nipype/interfaces/dtitk/registration.py

```diff
@@ -1,16 +1,28 @@
-from ..base import TraitedSpec, CommandLineInputSpec, traits, isdefined
+# -*- coding: utf-8 -*-
+# emacs: -*- mode: python; py-indent-offset: 4; indent-tabs-mode: nil -*-
+# vi: set ft=python sts=4 ts=4 sw=4 et:
+"""DTITK registration interfaces
+
+    Change directory to provide relative paths for doctests
+    >>> import os
+    >>> filepath = os.path.dirname( os.path.realpath( __file__ ) )
+    >>> datadir = os.path.realpath(os.path.join(filepath, '../../testing/data'))
+    >>> os.chdir(datadir)
```

Comment:
You can remove these prefatory lines. Tests are now run in that directory through another mechanism. (You may need to merge master to make it work.)

Similarly at the tops of other files.